packit_service/worker/handlers/copr.py

```diff
@@ -587,6 +591,16 @@ def handle_scan(self):
                 ),
             )
 
+    @staticmethod
+    def parse_dict_from_output(output: str):
+        json_pattern = r"\{.*?\}"
```

Comment:
Why do you need this? Why it can not be as simple as the [CLI code](https://github.com/packit/packit/blob/2e75e6ff4c0cadb55da1c8daf9315e4b0a69e4a8/packit/cli/scan_in_osh.py#L81-L83)?

Reply:
currently it is the same as the CLI code, but it raises a `JSONNDecodeError: ExtraData` ([see](https://red-hat-it.sentry.io/issues/5646624565/?project=1767823&query=is%3Aunresolved+issue.priority%3A%5Bhigh%2C+medium%5D&referrer=issue-stream&statsPeriod=30d&stream_index=0&utc=true))

as I write in the description, after looking into logs I see the output actually includes also some other info, mostly looks like Kerberos processing info

Reply:
Are you sure about that? `init_kerberos_ticket()` should run in a separate process, it can't affect stdout of `osh-cli`.

Reply:
yes I know, I think it is not about initialising the Kerberos ticket itself, I was coming from these logs:

```
2024-07-26T10:33:43.051834226+00:00 stderr F [2024-07-26 10:33:43,051: INFO/MainProcess] task.run_copr_build_end_handler[f1824d81-6a07-48e3-9ad1-d9ec6171e7d6] Scan submitted successfully, output: [41] 1721990020.122452: ccselect module realm chose cache DIR::/home/packit/kerberos/tkt with client principal packit-stg@FEDORAPROJECT.ORG for server principal HTTP/openscanhub.fedoraproject.org@FEDORAPROJECT.ORG
2024-07-26T10:33:43.051834226+00:00 stderr F [41] 1721990020.122454: Retrieving packit-stg@FEDORAPROJECT.ORG -> krb5_ccache_conf_data/start_realm@X-CACHECONF: from DIR::/home/packit/kerberos/tkt with result: -1765328243/Matching credential not found (filename: /home/packit/kerberos/tkt)
```
etc

Reply:
That's still suspicious to me, I'm not against the workaround for the time being, but I think we should investigate and fix this.